app/jobs/audit_manager.rb

```diff
@@ -0,0 +1,14 @@
+class AuditManager
+	def self.system_audit
+		packages = Package.stored
+		audit = Audit.new(user: User.system_user, packages: packages.count)
+		start_audit(user: User.system_user, audit: audit, packages: packages)
+	end
+
+	def self.start_audit (user, audit, packages)
```

Comment:
I would not pass the packages here. We're implying that it should be an audit of everything, so if we do the lookup of `Package.stored` here and update the audit with the count before it is saved, we can drop the parameter and duplicate lookups.